doc/rtd/topics/debugging.rst

```diff
@@ -163,3 +163,63 @@ commandline:
 
 Inspect cloud-init.log for output of what operations were performed as a
 result.
+
+.. _proposed_sru_testing:
+
+Manually verifying cloud-init Stable Release Updates (SRU)
```

Comment:
The overview sentence for this doc is:

> This topic will discuss general approaches for test and debug of cloud-init on deployed instances.

That's already borderline false, but is definitely no longer accurate with this section included. Could we revise it?

Reply:
retitled and added an intro paragraph